modules/AlphaWalletFoundation/AlphaWalletFoundation/EtherClient/TransactionSigning.swift

```diff
@@ -4,11 +4,11 @@ import BigInt
 import CryptoSwift
 
 public struct EthereumSignature {
-    public let v: UInt8
-    public let r: Data
-    public let s: Data
+    public let v: BigUInt
```

Comment:
Do you want to just change `v` to `BigUInt` like you did, but keep `r` and `s` as `Data`? Seems unnecessary to change? Small diff and better performance too